src/utils/filterTemplateProps/filterTemplateProps.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+import type { TProps } from '../parsePropName'
+import isTemplateProp from '../isTemplateProp'
+import pick from '../pick'
+
+export default function findTemplateProps (props: TProps) {
```

Comment:
Can we write it to be:
```js
const filterTemplateProps = pick('template')

export default filterTemplateProps
```

That would save us extra steps in `findTemplateProps`, when all it does it taking a prop starting from the name, which `pick` does, if I get it correctly.